### PR TITLE
Disable the "Back to top" button in the documentation

### LINKED ISSUE #12685: Can we disable the "Back to top" button in the docs?
It's constantly obstructing part of the content I want to read – I like reading lines close to the bottom of my screen, don't ask me why, I assume my vision is slightly better in this area. Anyway, it's super irritating this thing constantly appears and disappears, based on if and how and in which direction I'm scrolling. FYI I've not used a mouse in like 15 years, I do everything with a trackpad, because it makes pixel-precise scrolling so easy – so I tend to scroll a lot, often in tiny increments. The "Back to top" link is super not great for me. And I've never ever used it – if I want to scroll to the top, I do a swift swipe on my trackpad and voila, top of the page.

Alternative proposal if we want to keep it:
Do not display it conditionally (i.e., always show it, never hide it), and ensure it doesn't overlap with content.

Thoughts on this?

Screenshot 2024-06-26 at 20 04 04

### PR DESCRIPTION
**Pull request by hoechenberger**
Closes #12685

### CODE REVIEW

**Comment by cbrnr** [comment]
The docs build is failing. It would be good to see how the docs look like without that button though.

**Comment by larsoner** [comment]
> It would be good to see how the docs look like without that button though.

The rendered docs are still available:

https://output.circle-artifacts.com/output/job/6b6ff620-ee6c-4671-ab7b-0f559f6dfecc/artifacts/0/html/index.html

When the build finishes with a Sphinx error the docs are usually still produced. (If it errors out before Sphinx gets started on the other hand, like during a download step, there are no artifacts to expose.)

File: doc/changes/devel/12688.other.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Disable the "Back to top" button in the documentation, by `Richar Höchenberger`_.
```
[the remark this answers is not in the document]
Comment (larsoner) [reply doc/changes/devel/12688.other.rst:1]:
```suggestion
Disable the "Back to top" button in the documentation, by `Richard Höchenberger`_.
```

**Comment by cbrnr** [comment]
Thanks @hoechenberger!

**Pull request merged**